go.mod

```diff
@@ -28,3 +28,5 @@ require (
 	google.golang.org/grpc v1.28.0 // indirect
 	gopkg.in/ini.v1 v1.55.0 // indirect
 )
+
+replace github.com/hyperledger-labs/fabric-private-chaincode/ercc => ./ercc
```

Comment:
What's the reason to do this?

Reply:
attestation.GetVerifier in the ercc/attestation package is only available when we build the provider.go. This leads to the problem that go vet complains that getVerifer is not defined in the package as it checks aginst the go-mod dependency. When pointing to ercc directly all checks pass when provider.go exists.

Reply:
hmm, but essentially means our module is kindof broken as it should build self-contained?  But then maybe this is only temporary and the generation of provider.go is eventually going away? (btw: i forgot, what was actually the reason that we used templates rather than go build tags which would side step the problem as all files would exist out-of-the-box?)

Reply:
We used the templates to control wether we want to use mock attestation or the real one. ERCC used to be built by fabric and therefore we had no chance to add extra build commands regarding what attestation we want to use, therefore we generated the provider.go before and supplied it when installing/building the chaincode.

I think with external launchers we can overcome this issue as we can control how ercc is built. So we can use build tags.

Reply:
We definitely could do that using external builder and might have to do that for even more fundamental reasons iff ercc relies on C-based crypto/attestation libraries.   Will probably require though to add a new chaincode-lang `fpc-ercc` or alike so the external builder can distinguish it from other go code and, correspondingly, also would need to be added to the peer-cli wrapper as a second package special case (with `fpc-c` being the first one ...)